dotnet/src/Connectors/Connectors.Memory.AzureAISearch/AzureAISearchFilterTranslator.cs

```diff
@@ -139,20 +142,13 @@ private void GenerateLiteral(object? value)
 
     private void TranslateMember(MemberExpression memberExpression)
     {
-        switch (memberExpression)
+        if (this.TryBindProperty(memberExpression, out var property))
         {
-            case var _ when this.TryBindProperty(memberExpression, out var property):
-                this._filter.Append(property.StorageName); // TODO: Escape
-                return;
-
-            // Identify captured lambda variables, inline them as constants
-            case var _ when TryGetCapturedValue(memberExpression, out var capturedValue):
-                this.GenerateLiteral(capturedValue);
-                return;
-
-            default:
-                throw new NotSupportedException($"Member access for '{memberExpression.Member.Name}' is unsupported - only member access over the filter parameter are supported");
+            this._filter.Append(property.StorageName); // TODO: Escape
```

Comment:
@roji @dmytrostruk I fail to understand why this PR was reviewed and merged before https://github.com/microsoft/semantic-kernel/pull/11676 which aims to solve the problem of escaping.

I did my best to ensure I review other people PRs first so everybody gets unblocked. As a result, I got none of my PRs reviewed, and all with merge conflicts. This is nice... not.

Reply:
Really sorry @adamsitnik, I've been in a tunnel, focusing on my things, and I've neglected your PRs (and forgot that this would cause a conflict... I'm really sorry about that, my bad. I'll go review it now.

Reply:
I can also take care of the conflict resolution too, as it's my fault.